rmgpy/qm/gaussian.py

```diff
@@ -44,68 +44,78 @@
 class Gaussian:
     """
     A base class for all QM calculations that use Gaussian.
-    
+
     Classes such as :class:`GaussianMol` will inherit from this class.
     """
 
-    inputFileExtension = '.gjf'
-    outputFileExtension = '.log'
+    input_file_extension = ".gjf"
+    output_file_extension = ".log"
 
-    executablesToTry = ('g16', 'g09', 'g03')
+    executablesToTry = ("g16", "g09", "g03")
```

Comment:
if we're going through all the code churn of running black on the file and changing ' to " etc., should we also change `executablesToTry` and `possibleDirs` (below) to snake_case?
(I'm not a huge far of the un-necessarry code churn of black).